buf/registry/module/v1beta1/tag_service.proto

```diff
@@ -92,31 +91,36 @@ message ListTagsResponse {
   repeated Tag tags = 2;
 }
 
-message CreateTagsRequest {
-  // An individual request to create a Tag.
+message SetTagsRequest {
```

Comment:
Why not just `UpdateTags`, in the same style as other RPCs?

Reply:
Because semantically `Update` infers that the Tag existed before, I think? This RPC creates or moves the tag.

Reply:
`CreateOrUpdateTags` then

Reply:
I think the idea behind this was that we don't care if it exists or not, just put it here. Without this, the flow is
1. `GetTags`
2. If Tag doesn't exist, call `CreateTag`; otherwise call `UpdateTags`.

I'm fine with either approach, especially because sync will _already_ call `GetTags` (to only sync tags that have changed).

Reply:
Are there any real downsides to the non-atomic approach?

Reply:
If we have:
- `Create` create a tag that previously didn't exist
- `CreateOrUpdate` creates any tags that previously didn't exist, updates any that previously existed

Then I'm happy. I think `Create` needs to stay because it's impossible to say "create this tag and I want to be sure I'm not moving the tag and potentially breaking someone".

Reply:
I'm cool either way. I'll close this one in favor of #32 then, and comment there.